.perlcriticrc

```diff
@@ -10,7 +10,6 @@ severity = brutal
 [ControlStructures::ProhibitUnreachableCode]
 [Modules::ProhibitAutomaticExportation]
 [Modules::ProhibitConditionalUseStatements]
-[RegularExpressions::ProhibitUnusedCapture]
```

Comment:
I didn't seem to be able to trigger this policy with a test so removing it for now.